docs/migration-v2.md

```diff
@@ -0,0 +1,71 @@
+---
+hide_title: true
+title: Migration to v2
+---
+
+# Migration to Remirror v2
+
+This guide lays out the required steps to migrate from Remirror v1 to Remirror v2.
+
+Consult the [announcement post](/blog/announcement-v2) to learn more about the v2 release.
+
+## Installing
+
+If you are using TypeScript in your project, please remove all `@types/prosemirror-*` dependencies from your `package.json` file, for example, `@types/prosemirror-model` or `@types/prosemirror-view`.
+
+After that, you can update all Remirror packages to the latest version. We recommend using the [`npm-check-updates`](https://www.npmjs.com/package/npm-check-updates) package to do this.
+
+```bash
+$ npx npm-check-updates --filter '/remirror|prosemirror/' --deep --upgrade --target greatest
+```
+
+The command above will update all Remirror packages to the latest version in your `package.json` file. Then you can install them with `npm install`, `yarn install`, or `pnpm install`.
+
+## Deduplicate dependencies
+
+Remirror is a monorepo with multiple packages, with which some package managers struggle nowadays. You could therefore end up with duplicate versions of the same package in your project. We recommend you to check your lockfile (usually `package-lock.json`, `yarn.lock` or `pnpm-lock.yaml`) if you have installed multiple versions of the same package. Look out especially for the following packages:
+
+- `prosemirror-state`
+- `prosemirror-model`
+- `@remirror/core`
+- `@remirror/pm`
+
+If you find any duplicate versions, the easiest way is to remove the lockfile and the `node_modules` directory, then install your dependencies again. However, this will update **all** packages in your project, and you might not want to do that.
+
+If you don't want to update all packages, you can only deduplicate the packages you need. Based on your package manager, you can choose different ways to do that. Here are the best practices for the most common package managers:
+
+- `npm`: run [`npm dedupe`](https://docs.npmjs.com/cli/v8/commands/npm-dedupe)
+- `yarn` v1: install and use a third-party tool [`yarn-deduplicate`](https://www.npmjs.com/package/yarn-deduplicate)
+- `yarn` v2 or later: run [`yarn dedupe`](https://yarnpkg.com/en/docs/cli/dedupe) in your project root
```

Comment:
```suggestion
- `yarn` v2 or later: run [`yarn dedupe`](https://yarnpkg.com/cli/dedupe) in your project root
```

The previous link is pointing to the Yarn v1 docs and this new link is for Yarn v2+. 

Yarn v1 also have `yarn dedupe` command but it's not very useful.